AudioEncode/CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
-set(HEADERS)
```

Comment:
I'd strongly advise for listing headers in the sources for every module. As we've seen with #411 , this improves IDE integration among other things. It's fine you don't want to deal with that in this PR, I can also take care of it afterwards

Reply:
Let's make it in another PR.